docs/src/manual/multi.md

```diff
@@ -68,3 +68,56 @@ periods = TwoLevel(3, [5, 5, 10], [repr, repr, repr], 365.0);
 ```
 
 ## TwoLevelTree structure
+
+If there is uncertainty at a strategic level, this can be incorporated using the [`TwoLevelTree`](@ref) 
+time structure. This structure is represented by a tree, with each node corresponding to a strategic period that contains an operational time structure.
+
+The following example demonstrates how to create a regular tree where each strategic 
+period spans 3 years and is represented by a week with daily resolution.  
+The second  argument to the [`regular_tree`](@ref) function specifies the number 
+of branches at each level of the tree.
+```@repl ts
+using TimeStruct
+operational = SimpleTimes(7, 1);
+two_level_tree = regular_tree(3, [3,2], operational; op_per_strat = 52);
+```
+
+![Illustration of TwoLevelTree](./../figures/two_level_tree.png)
```

Comment:
```suggestion
![Illustration of `TwoLevelTree` with branch probabilities](./../figures/two_level_tree.png)
```
Not really necessary, but I think it is always good to have the complete information in a caption.

Reply:
Thinking about it again, seems we do not have that as caption?

Reply:
I do not know of any standard way of doing captions/titles in markdown. I have seen the syntax ![alt text](image url "title"), but it does not seem to work.